### ISSUE TITLE
Don't search for java command in legacy locations

### ISSUE THREAD

**Issue by mizdebsk** [fedora-java/javapackages#121]
Currently the `java` command is searched in the following locations under JAVA_HOME:
- `jre/sh/java` - old IMB Java, at least on AIX
- `jre/bin/java` no longer relevant singe Jigsaw, Java 9
- `bin/java` - still used

`bin/java` is the only one currently used. Therefore Javapackages should look for `java` command only under `${JAVA_HOME}/bin/java` and ignore the other two locations.

**Comment by fridrich** [comment]
This will not work for me here if there is only java-1_8_0-openjdk package installed. I have some users that insist using that on some systems :)

**Comment by fridrich** [comment]
OK, if there is no other java-N-openjdk installed, it would work through the /usr/bin/java alternative. But in other situations, it would mean that they have to change their scripts to set it to the jre subdir.

**Comment by mizdebsk** [comment]
Right, this is scoped for next major release (7.0.0) when we can assume that Java 8 is no longer relevant. On older systems javapackages 6.x can still be used.
